docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
The rules for image names are actually a bit more restrictive. Here's what I wrote up in #22264:

> A repository name is made up of slash-separated
> name components, optionally prefixed by a DNS hostname. The hostname
> must follow comply with standard DNS rules, but may not contain
> <code>_</code> characters. If a hostname is present, it may optionally
> be followed by a port number in the format <code>:8080</code>.
> Name components may contain lowercase characters, digits, and
> separators. A separator is defined as a period, one or two underscores,
> or one or more dashes. A name component may not start or end with
> a separator.